### PR TITLE
test(vagrantbox): automate vagrantbox tests CFY-3996

### PR DESCRIPTION
**Pull request by guy-mograbi-at-gigaspaces**

### CODE REVIEW

File: quickstart/automated-test/jenkins/script.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        echo "skip clean env.."
+    fi
+    vagrant destroy -f || echo "vagrant destroy failed, assuming machine is not up"
+}
+
+install_node
+clean_env
+
+VAGRANTFILE_URL="http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/${VERSION}/${PRERELEASE}-RELEASE/Vagrantfile"
+wget -O Vagrantfile $VAGRANTFILE_URL
```
Comment (m1keil) [inline quickstart/automated-test/jenkins/script.sh:57]: use `vagrant box add` instead

**Comment by guy-mograbi-at-gigaspaces** [comment]
need to fix some stuff - and then I will reopen

File: quickstart/automated-test/jenkins/script.sh (hunk 1 above)
Comment (guy-mograbi-at-gigaspaces) [inline quickstart/automated-test/jenkins/script.sh:57]: how can I use `vagrant box add` instead here?

Comment (guy-mograbi-at-gigaspaces) [inline quickstart/automated-test/jenkins/script.sh:57]: also please note that this file follows instructions from quickstart which says download and run vagrant up

http://getcloudify.org/guide/3.2/quickstart.html

Comment (m1keil) [inline quickstart/automated-test/jenkins/script.sh:57]: my bad, didn't notice this was vagrantfile you downloading

**Pull request closed without merging**